charts/dapr/values.yaml

```diff
@@ -57,8 +57,11 @@ global:
   #  placement:
   #  operator:
   #  injector:
+  
```

Comment:
Please revert changes to this file since they are just comments - will avoid merge conflicts